vars/chunkyTemplate.groovy

```diff
@@ -14,8 +14,7 @@ def call(Map parameters = [:], body) {
     def cloud = flow.getCloudConfig()
 
     def utils = new io.fabric8.Utils()
-    // 0.13 introduces a breaking change when defining pod env vars so check version before creating build pod
-    if (utils.isKubernetesPluginVersion013()) {
+
         if (utils.isUseOpenShiftS2IForBuilds()) {
```

Comment:
did you miss correcting the indentation ?

Reply:
@sthaha That's intentional to make the patch readable. Now its just red and its obvious what I did. I'll push another patch which fixes the indentation but not change anything else.